src/main/java/org/prebid/server/proto/openrtb/ext/request/tripleliftnative/ExtImpTripleliftNative.java

```diff
@@ -0,0 +1,21 @@
+package org.prebid.server.proto.openrtb.ext.request.tripleliftnative;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Value;
+
+import java.math.BigDecimal;
+
+/**
+ * Defines the contract for bidRequest.imp[i].ext.triplelift_native
+ */
+@AllArgsConstructor(staticName = "of")
+@Value
+public class ExtImpTripleliftNative {
```

Comment:
I don't think we should add a new model if the one for original `Triplelift` is the same. In GO they reused it as well.
Let's add it whenever these two bidders (one bidder actually...) would really need two different `imp.ext` models.